src/libraries/Common/src/Interop/FreeBSD/Interop.Process.cs

```diff
@@ -65,7 +65,7 @@ internal static unsafe int[] ListAllPids()
             }
             finally
             {
-                Marshal.FreeHGlobal((IntPtr)entries);
+                NativeMemory.Free(entries);
```

Comment:
Took this as an opportunity to switch to NativeMemory API and avoid the casts in these files.